_maps/map_files/cerestation/cerestation.dmm

```diff
@@ -84287,6 +84307,16 @@
 	icon_state = "neutralfull"
 	},
 /area/station/hallway/primary/port/east)
+"urK" = (
+/obj/machinery/mineral/stacking_machine{
+	stack_amt = 10;
```

Comment:
Any idea why we varedit to limit it to 10? it was the case with the cyberiad one too and you removed it.

Reply:
I just copied the one from Cyberiad - I didn't know why it was varedited to only limit it to 10 and I don't think I removed it on Cyberiad.

I'll probably remove the 10 stack limit - The recycler doesn't need more things limiting it really.

Reply:
The original amount is 50, i always felt like 10 was too small, feel free to remove it?

Reply:
Stacking machine on Farragus back to 50.